dts/bindings/display/ilitek,ili9xxx-common.yaml

```diff
@@ -4,7 +4,7 @@
 
 description: ILI9XXX display controllers common properties.
 
-include: spi-device.yaml
+include: [spi-device.yaml, display-controller.yaml]
```

Comment:
This seems to break bisectability by adding required `height` and `width` properties to the following bindings without updating in-tree users:

- dts/bindings/display/ilitek,ili9340.yaml
- dts/bindings/display/ilitek,ili9488.yaml

The dts/bindings/display/ilitek,ili9341.yaml binding is also affected, but doesn't seem to have any in-tree users.

Reply:
yes, realized it later, I will do better.